addons/mail/static/src/components/attachment_card/attachment_card.xml

```diff
@@ -4,6 +4,7 @@
         <div>
             <t t-if="attachmentCard">
                 <div class="o_AttachmentCard d-flex o-has-card-details"
+                    t-attf-class="{{ classname }}"
```

Comment:
I think we prefer having t-attf after t-att. (and at some point but it's probably fixed now there was a bug if the order was wrong).